client/overview/index.js

```diff
@@ -20,7 +20,7 @@ import ErrorBoundary from 'components/error-boundary';
 import TaskList from './task-list';
 import { getTasks, taskSort } from './task-list/tasks';
 import InboxNotifications from './inbox-notifications';
-import { useDisputes } from 'data';
+import { useDisputeStatusCounts } from 'data';
```

Comment:
```suggestion
import { useDisputeStatusCounts } from 'data';
import { disputeAwaitingResponseStatuses } from 'wcpay/disputes/filters/config';
```

To fix the re-rendering error, in combination with the suggestion below.